src/ButtonStrip/helper.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { Button } from '../Button'
+
+export const buttonArrayPropType = (props, propName, componentName) => {
+    const prop = props[propName]
+
+    let error = null
+    let count = 0
```

Comment:
can be:

```js
if (React.Children.count(prop) < 2) {
    new Error(`${componentName} should have at least 2 children.`)
}
```

Reply:
Thanks for that. I looked at that method in the docs this morning, but it hadn't clicked in my head that the `children` in that example was equivalent to the `pop` in our case. DUH.